ios/Podfile.lock

```diff
@@ -937,17 +937,17 @@ SPEC CHECKSUMS:
   React-logger: 933f80c97c633ee8965d609876848148e3fef438
   react-native-cameraroll: 60ac50a5209777cbccfe8d7a62d0743a9da87060
   react-native-config: 6502b1879f97ed5ac570a029961fc35ea606cd14
-  react-native-document-picker: 772d04a4bc5c35da9abe27b08ac271420ae3f9ef
+  react-native-document-picker: 429972f7ece4463aa5bcdd789622b3a674a3c5d1
   react-native-flipper: cd9eabd8917104c1bbdca2621717cdca3b2addef
   react-native-image-picker: ae1202414bd5c37c00b2a701daa5b6194a06b7d9
   react-native-netinfo: ebbcd8fbe1a0ce7035e43cd18c5a545dcb93dd08
   react-native-pdf: 4b5a9e4465a6a3b399e91dc4838eb44ddf716d1f
-  react-native-performance: 8edfa2bbc9a2af4a02f01d342118e413a95145e0
-  react-native-plaid-link-sdk: 9e0ebdaed648a237b36d5f6f6292b5147af92da7
-  react-native-progress-bar-android: ce95a69f11ac580799021633071368d08aaf9ad8
-  react-native-progress-view: 5816e8a6be812c2b122c6225a2a3db82d9008640
-  react-native-safe-area-context: 01158a92c300895d79dee447e980672dc3fb85a6
-  react-native-webview: 77ee909f73e1fcab76380f7dcc3344771fe61bd8
+  react-native-performance: 6bd6cfac80594775fb782405fceaaf206becf53b
+  react-native-plaid-link-sdk: 6ce5b2453b6a09ba3e975975c6a3fc7013a064fe
+  react-native-progress-bar-android: be43138ab7da30d51fc038bafa98e9ed594d0c40
+  react-native-progress-view: 21b1e29e70c7559c16c9e0a04c4adc19fce6ede2
+  react-native-safe-area-context: 79fea126c6830c85f65947c223a5e3058a666937
+  react-native-webview: 380c1a03ec94b7ed764dac8db1e7c9952d08c93a
```

Comment:
Are these changes necessary for the PR, as I see there are no updates on the libraries in package.json?

Reply:
I copied and pasted `Podfile.lock` from Expensify App repo a few minutes ago.